docs/campaign.rst

```diff
@@ -117,9 +117,15 @@ All methods are being used:
 cluster
 ~~~~~~~
 If value is "slurm", then the network ``nodes`` is filled based on the output
-of the ``info`` command. A tag will be also added for every 
+of the ``info`` command. A tag will be also added for every
 (partition, feature) tuple formatted like this: ``{partition}_{feature}``.
 
+slurm_blacklist_states
+~~~~~~~~~~~~~~~~~~~~~~
+List of SLURM node states used to filter-out nodes when ``cluster`` option
+is set to ``nodes``. Default states are down, drained, draining, error,
+fail, failing, future, maint, and reserved.
```

Comment:
drained, draining?

Reply:
### DRAINED
The node is unavailable for use per system administrator request. See the update node command in the scontrol(1) man page or the slurm.conf(5) man page for more information.

### DRAINING
The node is currently executing a job, but will not be allocated to additional jobs. The node state will be changed to state DRAINED when the last job on it completes. Nodes enter this state per system administrator request. See the update node command in the scontrol(1) man page or the slurm.conf(5) man page for more information.

Reply:
damn it, i just wanted to say that you may have forgotten those two, but i missed them in the list *sigh*...